src/main/java/org/broadinstitute/hellbender/utils/Utils.java

```diff
@@ -529,7 +532,29 @@ public static String calcMD5(final byte[] bytes) {
      * @throws IOException if the file could not be read
      */
     public static String calculateFileMD5( final File file ) throws IOException{
```

Comment:
Can you add a warning to the javadoc for this method that it slurps the entire file into memory, and should not be used for large files.

Reply:
Done!